### PR TITLE
docs: openapi spec

### PR DESCRIPTION
**Pull request by juliusmarminge**

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -4,4 +4,4 @@
 save-workspace-protocol=false
 prefer-workspace-packages=true
 
-engine-strict=true
+engine-strict=false
```
Comment (juliusmarminge) [inline .npmrc:7]: scalar got engine.node = 20 🤷🏼‍♂️ 

maybe we should update this repo to also use 20? it's the LTS afterall

Comment (markflorkowski) [reply .npmrc:7]: Now that it is supported on Vercel, sgtm 👍

File: docs/src/components/scalar.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import { useEffect } from "react";
+import { ApiReference as VueComponent } from "@scalar/api-reference";
+import { useTheme } from "nextra-theme-docs";
+import { applyVueInReact } from "veaury";
+
+const ApiReference = applyVueInReact(VueComponent);
+
+const specUrl =
+  "https://uploadthing-git-swagger-pinglabs.vercel.app/openapi.json";
```
Comment (markflorkowski) [inline docs/src/components/scalar.tsx:9]: Should point to prod url

**Review by markflorkowski — APPROVED** [review]

File: .npmrc (hunk 1 above)
Comment (juliusmarminge) [reply .npmrc:7]: #557

File: docs/src/components/scalar.tsx (hunk 2 above)
Comment (juliusmarminge) [reply docs/src/components/scalar.tsx:9]: yea, need this merged first: https://github.com/pingdotgg/uploadthing-infra/pull/283

File: packages/uploadthing/src/sdk/index.ts [hunk 3]
```diff
@@ -302,7 +302,7 @@ export class UTApi {
     if (!Array.isArray(updates)) updates = [updates];
 
     return this.requestUploadThing<{ success: true }>(
-      "/api/renameFile",
+      "/api/renameFiles",
```
Comment (juliusmarminge) [inline packages/uploadthing/src/sdk/index.ts:305]: minor consistency changed now that we've "aliased" this on infra side

**Pull request merged**